README.md

```diff
@@ -22,7 +25,7 @@ PerfCompare is hosted on heroku, and is updated every time commits are pushed to
 ## Setup
 
 ### Requirements
-
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
```

Comment:
This looks like an unrelated change, can you rebase your branch on master?

Reply:
@jrbenny35 I have rebased.